src/Core/Localization/CLDR/Reader.php

```diff
@@ -503,7 +503,7 @@ protected function mapLocaleData(SimpleXMLElement $xmlLocaleData, $supplementalD
                 $currencyData->setIsoCode($currencyCode);
 
                 // check if currency is still active in one territory
-                $currencyDates = $this->supplementalXml->supplementalData->xpath('//region/currency[@iso4217="' . $currencyCode . '"]');
+                $currencyDates = $this->supplementalXml->currencyData->xpath('//region/currency[@iso4217="' . $currencyCode . '"]');
```

Comment:
Ping @jolelievre

Reply:
Seems like you remove the first node `supplementalData` from the XML here https://github.com/PrestaShop/PrestaShop/blob/7ee63784e2a1cba1322206962c41eebe47491c69/localization/CLDR/core/common/supplemental/supplementalData.xml#L9

Does it mean the `simplexml_load_file` returns is different in PHP 8.0? Or does it mean it was never right? 😅

Reply:
It was never right 😅